package.json

```diff
@@ -26,7 +26,8 @@
     "build:js:edit": "node ./notebook/build.js edit/js/main.js edit/js/main.bundle.js",
     "build:js:tree": "node ./notebook/build.js tree/js/main.js tree/js/main.bundle.js",
     "build:js:auth": "node ./notebook/build.js auth/js/main.js auth/js/main.bundle.js",
-    "build:js:terminal": "node ./notebook/build.js terminal/js/main.js terminal/js/main.bundle.js"
+    "build:js:terminal": "node ./notebook/build.js terminal/js/main.js terminal/js/main.bundle.js",
+    "prepublish": "npm run bower && npm run build"
```

Comment:
It should be okay, that behavior only happens on local `npm install` - it doesn't happen on `npm install jupyter-notebook-deps --save` or save-dev for example.  https://docs.npmjs.com/misc/scripts